app/controllers/candidate_interface/sign_in_controller.rb

```diff
@@ -24,12 +24,17 @@ def create
 
     def authenticate
       candidate = FindCandidateByToken.call(raw_token: params[:token])
+      flash[:success] = 'Your apply for teacher training account has been created' if candidate&.last_signed_in_at.nil?
```

Comment:
This can be moved inside the `if candidate` so we don't have to add the `&`.

Reply:
thanks 👍